src/client/v2/algod/models/types.ts

```diff
@@ -1301,15 +1317,15 @@ export class ApplicationStateSchema extends BaseModel {
     data: Record<string, any>
   ): ApplicationStateSchema {
     /* eslint-disable dot-notation */
-    if (typeof data['num-uint'] === 'undefined')
-      throw new Error(`Response is missing required field 'num-uint': ${data}`);
     if (typeof data['num-byte-slice'] === 'undefined')
       throw new Error(
         `Response is missing required field 'num-byte-slice': ${data}`
       );
+    if (typeof data['num-uint'] === 'undefined')
+      throw new Error(`Response is missing required field 'num-uint': ${data}`);
```

Comment:
Why are these required, rather than defaulting to 0?

Reply:
The REST API declares them as required

Reply:
Required in the _response_? (I'm judging based on the error string.)

Seems odd that the SDK doesn't trust server side output in this one particular way.